content/4_Gold/Modular.mdx

```diff
@@ -81,6 +81,13 @@ multiply as normal.
 	>
 		some practice probs
 	</Resource>
+    <Resource
+        source="MONT@Modern Olympiad Number Theory"
+        title="Chapters 2 and 5 - Basic to Advanced Modular Arithmetic"
```

Comment:
same comment as for other module. maybe

`2, 5 - Modular Arithmetic`

Reply:
done

Reply:
title hasn't changed?

Reply:
wait let me see

Reply:
now?